Tests/UnitTests/Core/TrackFinding/CombinatorialKalmanFilterTests.cpp

```diff
@@ -370,8 +395,11 @@ BOOST_AUTO_TEST_CASE(ZeroFieldBackward) {
 
   Fixture::TestSourceLinkAccessor slAccessor;
   slAccessor.container = &f.sourceLinks;
-  options.sourceLinkAccessor.connect<&Fixture::TestSourceLinkAccessor::range>(
-      &slAccessor);
+
+  auto trackStateCreator = makeTrackStateCreator(slAccessor, f.measSel);
```

Comment:
Is this used anywhere?

Reply:
Yes, that is the delegate for extendOrBranchTrajectory. Maybe I should rename it ?

Reply:
name now agrees with the delegate name.